components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -33,36 +34,42 @@
             <chef-button primary (click)="openCreateModal()">Add Chef Server</chef-button>
           </app-authorized>
         </chef-toolbar>
-        <chef-table>
-          <chef-thead>
-            <chef-tr>
-              <chef-th  class="no-border-th">Name</chef-th>
-              <chef-th class="no-border-th">FQDN</chef-th>
-              <chef-th class="no-border-th">IP Address</chef-th>
-              <chef-th class="no-border-th">No Of Orgs</chef-th>
-              <chef-th class="three-dot-column  no-border-th"></chef-th>
-            </chef-tr>
-          </chef-thead>
-          <chef-tbody>
-            <chef-tr *ngFor="let server of sortedChefServers$ | async">
-              <chef-td>
-                <a [routerLink]="['/infrastructure/chef-servers', server.id]">{{ server.name }}</a>
-              </chef-td>
-              <chef-td>{{ server.fqdn }}</chef-td>
-              <chef-td>{{ server.ip_address }}</chef-td>
-              <chef-td>{{ server.orgs_count }}</chef-td>
-              <chef-td class="three-dot-column">
-                <!-- <app-authorized [allOf]="['/api/v0/infra/servers/{id}', 'delete', server.id]"> -->
-                  <mat-select panelClass="chef-control-menu" id="menu-{{server.id}}">
-                    <mat-option (onSelectionChange)="startServerDelete($event, server)" data-cy="remove-server">Remove
-                      Chef Server</mat-option>
-                  </mat-select>
-                <!-- </app-authorized>   -->
-              </chef-td>
-            </chef-tr>
-          </chef-tbody>
-        </chef-table>
+        <div *ngIf="servers.length != 0">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th class="no-border-th">Name</chef-th>
+                <chef-th class="no-border-th">FQDN</chef-th>
+                <chef-th class="no-border-th">IP Address</chef-th>
+                <chef-th class="no-border-th">No Of Orgs</chef-th>
```

Comment:
Not from you, I understand, but let's fix this--"No" is a poor abbreviation:
```suggestion
                <chef-th class="no-border-th">Number Of Orgs</chef-th>
```